client/src/rpb/rpb.scss

```diff
@@ -6,22 +6,29 @@
   min-width: 75%;
 }
 
+.modal-container {
```

Comment:
Could this be `rpb-modal-container`? Just to keep things scoped. Fine right now, but feels like the kind of class name that someone will accidentally step on eventually.